packages/geo-diary-v2/src/graphql/client.ts

```diff
@@ -14,11 +15,22 @@ export const generateRequest = (accessToken: string) => async (operation: Operat
 
 export const onError: ErrorHandler = ({ graphQLErrors, networkError }: ErrorResponse) => {
   if (graphQLErrors) {
-    graphQLErrors.map(({ message, locations, path }) =>
-      console.log(`[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`),
-    )
+    graphQLErrors.map(({ message, locations, path }) => {
+      console.log(`[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`)
+      const messageNotIncludeTraceID = message?.split('-')?.[1]
+      notification.error({
+        message: messageNotIncludeTraceID,
+        placement: 'bottomRight',
+      })
+    })
+  }
+  if (networkError) {
+    console.log(`[Network error]: ${networkError}`)
```

Comment:
Can we remove two `console.log`?

Reply:
@vuhuucuong I think it should console.error instead